owned-manifests/clusterapi-manager-controllers.yaml

```diff
@@ -61,6 +61,20 @@ spec:
         args:
           - --logtostderr=true
           - --v=3
+      - name: nodelink-controller-deprecated
+        image: quay.io/coreos/machine-api-operator:origin-v4.0-2019-01-31-041134
```

Comment:
I think it would be nice to template these in after having read them from the config like we do for the others. Even if it's temporary, it would be nice for these to come from the release image. I suppose this is not a total blocker though.

Reply:
this is never gonna come from the payload/release image. The ones coming from the payload are the ones built from master so adding support for the new types.
The payload CI build will replace every appearance of https://github.com/openshift/machine-api-operator/blob/master/install/image-references#L8 with the corresponding image Stream built by CI.
We could template in the configMap though but it wouldn't make any difference